server/src/websocket_uploader/engine_uploader.c

```diff
@@ -394,36 +409,35 @@ void load_cached_verifier_url() {
 			
 			free(buffer);
 			syslog_server(LOG_INFO, 
-			              "Websocket load cached verifier url: loaded %s from cache", 
+			              "Load cached verifier URL - loaded URL %s from cache", 
 			              verifier_url);
 		} else {
-			syslog_server(LOG_NOTICE, 
-			              "Websocket load cached verifier url: no content in cache file");
+			syslog_server(LOG_NOTICE,  "Load cached verifier URL - no content in cache file");
 		}
 	} else {
 		syslog_server(LOG_NOTICE, 
-		              "Websocket load cached verifier url: url cache file could not be opened");
+		              "Load cached verifier URL - cache file could not be opened");
 	}
 }
 
 
 void cache_verifier_url() {
 	// write current verifier url to cache file unless url is null
 	if (verifier_url == NULL) {
-		syslog_server(LOG_NOTICE, "Websocket cache verifier url: url not cached as it is NULL");
+		syslog_server(LOG_NOTICE, "Cache verifier URL - not cached as URL is NULL");
 		return;
 	}
 	
 	// Open file, write string into file, close file.
 	FILE* file = fopen(cache_file_verifier_url, "w");
 	if (file == NULL) {
-		syslog_server(LOG_ERR, "Websocket cache verifier url: file opening failed");
+		syslog_server(LOG_ERR, "Cache verifier URL - cache file opening failed");
```

Comment:
Could be interpreted as: the caching of "file opening" failed.
Suggestion: "failed to open cached file"